_sass/components/_sticky_nav.scss

```diff
@@ -15,7 +15,10 @@
 .sticky_nav-nav_item {
   list-style-type: none;
   padding: $base-padding-lite;
+  @include heading('para-lg');
```

Comment:
Expected item on line 18 to appear before line 16. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets